tests/rs_sqla_test_utils/models.py

```diff
@@ -14,6 +14,14 @@ class Basic(Base):
     )
 
 
+class BasicInSchema(Base):
+    __tablename__ = 'schema.basic'
+    col1 = sa.Column(
+        sa.Integer(), primary_key=True,
+        info={'distkey': True, 'sortkey': True}
+    )
```

Comment:
Thanks.

Should I update the PR? I think it might be easier for someone of you to do it, since at the moment I don't have any personal test redshift cluster, and so I cannot run this code.

(again, the fact that the schema might be the issue is only my hypothesis)